dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -29,33 +29,40 @@
-                    <div :class="`p-badge badge-${category.color}`">
-                      <span>{{ category.label.toUpperCase() }}</span>
+                  <template v-if="subcategoryVisibility.get(subcategory) ?? true">
+                    <div class="col-3 p-3 topicLabel">
+                      <h4 :id="subcategory.name" class="anchor title">{{ subcategory.label }}</h4>
+                      <div :class="`p-badge badge-${category.color}`">
+                        <span>{{ category.label.toUpperCase() }}</span>
+                      </div>
                     </div>
-                  </div>
 
-                  <div class="col-9 formFields">
-                    <FormKit v-for="field in subcategory.fields" :key="field" type="group" :name="subcategory.name">
-                      <component
-                        v-if="field.showIf(companyAssociatedLksgData.data)"
-                        :is="field.component"
-                        :label="field.label"
-                        :placeholder="field.placeholder"
-                        :description="field.description"
-                        :name="field.name"
-                        :options="field.options"
-                        :required="field.required"
-                        :certificateRequiredIfYes="field.certificateRequiredIfYes"
-                        :validation="field.validation"
-                        :validation-label="field.validationLabel"
-                        :data-test="field.name"
-                        @documentUpdated="updateDocumentList"
-                        :ref="field.name"
-                      />
-                    </FormKit>
-                  </div>
+                    <div class="col-9 formFields">
+                      <FormKit v-for="field in subcategory.fields" :key="field" type="group" :name="subcategory.name">
+                        <component
+                          v-if="field.showIf(companyAssociatedLksgData.data)"
+                          :is="field.component"
+                          :label="field.label"
+                          :placeholder="field.placeholder"
+                          :description="field.description"
+                          :name="field.name"
+                          :options="field.options"
+                          :required="field.required"
+                          :certificateRequiredIfYes="field.certificateRequiredIfYes"
+                          :validation="field.validation"
+                          :validation-label="field.validationLabel"
+                          :data-test="field.name"
+                          @documentUpdated="updateDocumentList"
+                          :ref="field.name"
+                          :existingFieldValues="
+                            field.existingFieldValues
+                              ? field.existingFieldValues(companyAssociatedLksgData.data)
+                              : undefined
```

Comment:
Reminder for myself to review the usage of this new `Field` property when dev on new frontend component is finished.

Reply:
This function is no longer there.

Reply:
Yes right. I will close this now.